go_router/test/go_router_test.dart

```diff
@@ -244,8 +237,8 @@ void main() {
       {
         final matches = router.routerDelegate.matches;
         expect(matches.length, 2);
-        expect(matches[0].subloc, '/');
```

Comment:
fyi, the `[0]` came from copy-pasting other test code where I was using `[1]` and somethings `[2]` as well.

Reply:
Yeah, this came up during the code review from @pq. It was easier to just find/replace this everywhere :)